core/system_menu_method.js

```diff
@@ -241,6 +242,23 @@ function nextArea(callingMenu, formData, extraArgs, cb) {
     );
 }
 
+function setClientEncoding(callingMenu, formData, extraArgs, cb) {
+    // TODO: Also add other encoding types
```

Comment:
In `client_term.js` the encoding property is checked, but it's just ignored and logged if a unsupported encoding is passed. Perhaps we should check it here, e.g.

```
iconv.encodingExists(...)
```

Reply:
I'd be happy to do that, but I'm not sure what to do here besides log?

Reply:
I think you're right, not much to do here. Since the client_term already checks, we can just keep it there and keep checking there as well.